### DISCUSSION TITLE
[Feature] Add support for rootless docker

### DISCUSSION THREAD

**Discussion by 89luca89** [89luca89/distrobox#1901] [Ideas]
Right now we have support for rootless podman, but docker only works in rootful mode.
This can create unpleasant situations like described in #201 

Right now rootless docker is not working like podman and using it prevents a lot of integration with the system, like use of `--network host`

This issue will keep track of the development of rootless docker and the possibility to integrate it

**Comment by catuhana** [comment]
What's the status of this?

**Comment by trofosila** [comment]
@89luca89 any progress on this?

**Comment by frafra** [comment]
I am able to use distrobox with rootless mode, but there is an error setting up the user:
```
OCI runtime exec failed: exec failed: unable to start container process: unable to setup user: invalid argument: unknown
```

Nonetheless, since the container root user is mapped as a regular user, this is not a problem, and I can enter into the container using `docker exec "$container" /bin/bash`.
